README.md

```diff
@@ -94,7 +94,7 @@ First, clone the application from the repo and create the local `.env` file:
 
 ```bash
 $ git clone https://github.com/Tandemly/eos-wallet-wip.git
-$ cd eos-wallet-aip
+$ cd eos-wallet-wip
```

Comment:
Good catch...both that and the line above it should actually change to represent the "eos-web-wallet" name now though.  I've made another PR to reflect that, so I'm going to close this one.  Thank you though!

Reply:
Thank you, this is will my first commit!